test/scrabble_test.rb

```diff
@@ -5,9 +5,54 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_empty_string
+    assert_equal 0, Scrabble.new.score("")
+  end
+
+  def test_it_can_score_nil
```

Comment:
I like your breakdown of testing- it's great to create separate tests for each edge case!